.bingo/junitreport.mod

```diff
@@ -2,7 +2,4 @@ module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
 go 1.14
```

Comment:
Does this need an update to 1.20?

Reply:
All our tests pass so doesn't look critical